code/game/machinery/vending_types.dm

```diff
@@ -94,6 +94,119 @@
 	)
 	product_ads = "Only the finest!;Have some tools.;The most robust equipment.;The finest gear in space!"
 
+/obj/machinery/vending/assist/ninja
+	name = "Ninja Module Vendor"
+	vend_id = "tools"
+	can_move = 0
+	can_tamper = 0
+	use_power = 0
+	emagged = 1
+	categories = CAT_NORMAL | CAT_HIDDEN
+	products = list()
+	contraband = list(
+		/obj/item/device/pda/fake = 1,
+		/obj/item/rig_module/actuators/combat = 1,
+		/obj/item/rig_module/device/healthscanner = 1,
+		/obj/item/rig_module/chem_dispenser/ninja = 1,
+		/obj/item/rig_module/power_sink = 1,
+		/obj/item/rig_module/emergency_powergenerator = 1,
+		/obj/item/rig_module/emag_hand = 1,
+		/obj/item/rig_module/vision/multi = 1,
+		/obj/item/weapon/material/sword/katana = 2,
+		/obj/item/weapon/material/sword/dao = 2,
+		/obj/item/weapon/material/star = 4,
+		/obj/item/weapon/material/twohanded/spear/steel = 1,
+		/obj/item/weapon/nullrod/nullstaff = 1,
+		/obj/item/weapon/cloaking_device/weak = 1,
```

Comment:
No, this item is terrible and should not ever be used again without someone fixing their bugs, it was removed from the traitor uplink for a reason.

Reply:
Wasn't aware of bugs. Will remove it.